main.py

```diff
@@ -0,0 +1,49 @@
+from flask import Flask, render_template, request, redirect
+from flask_mysqldb import MySQL
+ 
+app = Flask(__name__)
+
+app.config['MYSQL_HOST'] = 'localhost'
+app.config['MYSQL_USER'] = 'root'
+app.config['MYSQL_PASSWORD'] = 'Shikhar@970'
+app.config['MYSQL_DB'] = 'mysqldb'
+
+mysql = MySQL(app)
+
+@app.route("/")
+def signup_page():
+    return render_template("signup.html")
+
+
+@app.route("/signup", methods=["POST"])
+def manager_page():
```

Comment:
Why do you think `manager_page` name will suit for this function name?

Reply:
At the beginning of this application, I focus on, finish the task within time, that's why I couldn't focus on the clean code. Definitely, I will write the meaningful function here.